packages/libsigcplucpluc.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Libsigcplusplus < Package
+  description 'http://libsigc.sourceforge.net/'
+  homepage 'libsigc++ implements a typesafe callback system for standard C++.'
```

Comment:
Looks like description and homepage are swapped.

Reply:
No. This is the official link from libsigc.sourceforge.net